### PR TITLE
Docs updates for APM PHP release 0.47.0

### PR DESCRIPTION
**Pull request by labbati**
**Blocked on release of [0.47.0](https://github.com/DataDog/dd-trace-php/releases)**

### What does this PR do?
Docs updates for APM PHP release `0.47.0`

### Preview link

[Preview](https://docs-staging.datadoghq.com/apm/php/0.47.0/tracing/setup/php/)

Check preview base path using the URL in details in `preview` status check.

### CODE REVIEW

File: content/en/tracing/setup/php.md [hunk 1]
```diff
@@ -251,7 +252,7 @@ DD_TRACE_DEBUG=true php -S localhost:8888
 | `DD_TRACE_DEBUG`                          | `false`     | Enable [debug mode](#custom-url-to-resource-mapping) for the tracer                                                                            |
 | `DD_TRACE_ENABLED`                        | `true`      | Enable the tracer globally                                                                                                                     |
 | `DD_TRACE_GENERATE_ROOT_SPAN`             | `true`      | Automatically generate a top-level span; set to `false` in conjunction with `DD_TRACE_AUTO_FLUSH_ENABLED=1` to trace long-running processes    |
-| `DD_TRACE_GLOBAL_TAGS`                    | `null`      | Tags to be set on all spans, for example: `key1:value1,key2:value2`.                                                                                 |
+| `DD_TAGS`                                 | `null`      | Tags to be set on all spans, e.g.: `key1:value1,key2:value2`.                                                                                  |
```
Comment (SammyK) [inline content/en/tracing/setup/php.md:255]: Tiny nit: All of the `e.g.`'s were converted to `for example` in #7620.

**Comment by apigirl** [comment]
hi! we recently updated our S3 strategy in staging, so I had to merge master into this PR for the staging link to continue to work. If you're working locally, you'll just need to pull in my changes.

**Comment by labbati** [comment]
It sounds good thanks @apigirl

File: content/en/tracing/setup/php.md (hunk 1 above)
Comment (sarina-dd) [reply content/en/tracing/setup/php.md:255]: +1, for example is to ensure appropriate translation :)

Comment (sarina-dd) [inline content/en/tracing/setup/php.md:255]:
```suggestion
| `DD_TAGS`                                 | `null`      | Tags to be set on all spans, for example: `key1:value1,key2:value2`.                                                                                  |
```

File: content/en/tracing/setup/php.md [hunk 2]
```diff
@@ -262,6 +263,7 @@ DD_TRACE_DEBUG=true php -S localhost:8888
 | `DD_TRACE_URL_AS_RESOURCE_NAMES_ENABLED`  | `true`      | Enable URL's as resource names (see [Map resource names to normalized URI](#map-resource-names-to-normalized-uri)).                            |
 | `DD_<INTEGRATION>_ANALYTICS_ENABLED`      | `false`     | A flag to enable app analytics for relevant spans in a specific integration (see [Integration names](#integration-names)).                       |
 | `DD_<INTEGRATION>_ANALYTICS_SAMPLE_RATE`  | `1.0`       | Set the app analytics sample rate for relevant spans in a specific integration (see [Integration names](#integration-names)).                  |
+| `DD_VERSION`                              | `null`      | Set an application’s version in traces and logs e.g. `1.2.3`, `6c44da20`, `2020.02.13`. Added in version `0.47.0`.                             |
```
Comment (sarina-dd) [inline content/en/tracing/setup/php.md:266]:
```suggestion
| `DD_VERSION`                              | `null`      | Set an application’s version in traces and logs, for example: `1.2.3`, `6c44da20`, `2020.02.13`. Added in version `0.47.0`.                             |
```

File: content/en/tracing/setup/php.md [hunk 3]
```diff
@@ -235,9 +235,10 @@ DD_TRACE_DEBUG=true php -S localhost:8888
 | `DD_AGENT_HOST`                           | `localhost` | The Agent host name                                                                                                                            |
 | `DD_AUTOFINISH_SPANS`                     | `false`     | Whether spans are automatically finished when the tracer is flushed                                                                            |
 | `DD_DISTRIBUTED_TRACING`                  | `true`      | Whether to enable distributed tracing                                                                                                          |
+| `DD_ENV`                                  | `null`      | Set an application’s environment e.g. `prod`, `pre-prod`, `stage`. Added in version `0.47.0`.                                                  |
```
Comment (morrisonlevi) [inline content/en/tracing/setup/php.md:238]:
```suggestion
| `DD_ENV`                                  | `null`      | Set an application’s environment, for example: `prod`, `pre-prod`, `stage`. Added in version `0.47.0`.                                            |
```

**Comment by morrisonlevi** [comment]
I made some changes; can I get this re-reviewed? I am preparing for the release of 0.47.0.

**Comment by SammyK** [comment]
0.47.0 was just released so this is ready for merge: https://github.com/DataDog/dd-trace-php/releases/tag/0.47.0

**Pull request merged**